src/wallet/rpcwallet.cpp

```diff
@@ -292,7 +292,7 @@ static UniValue setlabel(const JSONRPCRequest& request)
 
 static CTransactionRef SendMoney(CWallet * const pwallet, const CTxDestination &address, CAmount nValue, bool fSubtractFeeFromAmount, const CCoinControl& coin_control, mapValue_t mapValue)
 {
-    CAmount curBalance = pwallet->GetBalance();
+    CAmount curBalance = pwallet->GetBalance(ISMINE_SPENDABLE, 0, coin_control.m_allow_dirty_addresses);
```

Comment:
I think this change should be in the _wallet/rpc: add allow_dirty option to sendtoaddress_ commit (not the _wallet/rpc: add include_dirty flag to getbalance command_ commit)

Reply:
Oops, yep. Thanks.